client/landing/gutenboarding/components/signup-form/index.tsx

```diff
@@ -43,6 +43,7 @@ const SignupForm = ( { onRequestClose }: Props ) => {
 	const makePath = usePath();
 	const currentStep = useCurrentStep();
 	const isMobile = useViewportMatch( 'small', '<' );
+	const isAnchorFmSignup = useSelect( ( select ) => select( ONBOARD_STORE ).getSelectedPodcastId() );
```

Comment:
Noting: isAnchorFmSignup sounds like a boolean, but it's a string with value `"2"` (when using Anchor), or `""` (empty string, when not using anchor). It does work, since empty string is a falsey value, it just wasn't what I expected.

Reply:
I agree. `!!` or `Boolean()` to make it explicit as a bool

Reply:
Added boolean selector here ac1e119fb2